src/components/dashboard/FeedList.js

```diff
@@ -128,13 +128,13 @@ const FeedList = ({
                 dialogShown: true,
               })
               userStorage.updateOTPLEventStatus(id, 'pending')
-              showErrorDialog('The payment could not be canceled at this time', 'CANCEL-PAYMNET-1')
+              showErrorDialog('The payment could not be canceled at this time. Please try again', ExceptionCode.E11)
```

Comment:
why decorate isn't used ? it adds Exx as the exception.code to be reported to Sentry

please see the other places where decorate is used

Reply:
The task requirement was to replace the technical error message with a general human-readable error message which won't confuse the users in places where the e.message is shown. And mark those places with Ex code.
Exactly this place already contain an understandable error message.

Reply:
Ok. I see. I used decorate for those places you asked as an error code should be sent to the sentry. See new commit